X509.xs

```diff
@@ -329,9 +329,7 @@ BOOT:
   }
 #if OPENSSL_VERSION_NUMBER < 0x10100000
   ERR_load_crypto_strings();
-# ifdef OPENSSL_LOAD_CONF
   OPENSSL_add_all_algorithms_conf;
```

Comment:
you need to put the parens back, here, otherwise the function is not called!

Reply:
Ah, I forgot about the parenthesis, done